pipeline/plugins/includes/tasks.py

```diff
@@ -136,3 +138,29 @@ def clear_temporary_directory(ssh_conn_id, tmpdir_root):
     ssh_hook = SSHHook(ssh_conn_id=ssh_conn_id)
     with ssh_hook.get_conn() as ssh_client:
         ssh_client.exec_command(f"rm -r {tmpdir_root}/*")
+
+
+@task(task_id="restic_backup", trigger_rule="all_done")
+def restic_backup(ssh_conn_id, script_path, output_dir):
+    ssh_hook = SSHHook(ssh_conn_id=ssh_conn_id)
+    with ssh_hook.get_conn() as ssh_client:
+        with open("/home/ubuntu/restic_env.yaml", "r") as fobj:
+            envs = yaml.load(fobj, Loader=yaml.FullLoader)
+        
+        envs_str = " ".join([f"export {key}={value};" for key, value in envs.items()])
+        print("Creating snapshot of downloaded images")
```

Comment:
I don't know does this matter, but we don't currently use `print()` much, but `logging` instead. (Though we do use `print()` in a few places too, mostly in the binding-fetching code). Is there a potential scenario where the inconsistency could bite us?

Reply:
Airflow seems to treat `print()`s as an INFO logging, so it's a lot more convenient do use print outside of operators, because the `self.log` comes with the `BaseOperator`. Not sure if there are any scenarios where `print()` couls cause issues.

Reply:
FWIW I don't see a problem here either.